hubble-be/src/main/java/com/baidu/hugegraph/entity/load/LoadTask.java

```diff
@@ -180,7 +182,11 @@ public void run() {
                     this.status = LoadStatus.FAILED;
                 }
             }
-            this.fileReadLines = this.context().newProgress().totalInputReaded();
+            long readLines = this.context().newProgress().totalInputReaded();
+            if (readLines == 0L) {
+                readLines = this.context().oldProgress().totalInputReaded();
```

Comment:
why set to old value when new value is 0

Reply:
no need.